packages/wasm-utils/src/note/mod.rs

```diff
@@ -766,6 +815,37 @@ impl JsNote {
 		self.mutate_index(index).map_err(|e| e.into())
 	}
 
+	#[wasm_bindgen(js_name = defaultUtxoNote)]
+	pub fn default_utxo_note(&self) -> Result<JsNote, OperationError> {
+		let mut new_note = self.clone();
+		let chain_id: u64 = self
+			.target_chain_id
+			.parse()
+			.map_err(|_| OpStatusCode::InvalidTargetChain)?;
+
+		let utxo = vanchor::generate_secrets(
+			0,
```

Comment:
Is this meant to be the chain_id? Recall that a default UTXO still needs to have the chain ID where it is being "spent". So this function will need to have access to the chain ID where the transaction is taking place.

Reply:
Yeah this is using the `traget_chain_id` value for the` chain_id` of the empty UTXO

Reply:
This is consuming the `target_chain_id` and using it as the value of the default UTXO `chain_id`
As for input, all UTXOs share the same `chain_id`